frontend/src/components/pages/admin/CreateAssessmentPage.tsx

```diff
@@ -40,117 +45,123 @@ const CreateAssessmentPage = (): React.ReactElement => {
+            control={control}
+            name="assessmentType"
+            rules={{ required: "Please select a type of assessment" }}
+            render={({
+              field: { onChange, value, name, ref },
+              fieldState: { error },
+            }) => (
+              <FormControl isRequired isInvalid={Boolean(error)}>
+                <FormLabel color="grey.400">Type of Assessment</FormLabel>
+                <RadioGroup
+                  name={name}
+                  ref={ref}
+                  onChange={onChange}
+                  value={value}
+                >
+                  <VStack align="left" spacing={0.5}>
+                    <Radio value="beginning">Beginning of Grade</Radio>
+                    <Radio value="end">End of Grade</Radio>
+                  </VStack>
+                </RadioGroup>
+                <FormErrorMessage>{error?.message}</FormErrorMessage>
+              </FormControl>
+            )}
+          />
+        </Box>
 
-          <FormControl isRequired isInvalid={Boolean(errors.region)}>
-            <FormLabel color="grey.400">Region</FormLabel>
-            <Input
-              {...register("region", { required: "Please enter a region" })}
+        <FormControl>
+          <Text textStyle="subtitle2" mb="2">
+            Curriculum
+          </Text>
+          <HStack width="100%">
+            <Controller
+              control={control}
+              name="country"
+              rules={{ required: "Please select a country" }}
+              render={({
+                field: { onChange, value, name },
+                fieldState: { error },
+              }) => (
+                <FormControl isRequired isInvalid={Boolean(error)} mr={2}>
+                  <FormLabel color="grey.400">Country</FormLabel>
+                  <Select
+                    name={name}
+                    onChange={onChange}
+                    value={value}
+                    options={countryOptions}
+                    placeholder=""
+                    useBasicStyles
+                  />
+                  <FormErrorMessage>{error?.message}</FormErrorMessage>
+                </FormControl>
+              )}
             />
-            <FormErrorMessage> {errors.region?.message} </FormErrorMessage>
-          </FormControl>
-        </HStack>
-      </FormControl>
 
-      <Button onClick={handleSubmit(onSubmit, onError)}>Submit</Button>
-    </VStack>
+            <FormControl isRequired isInvalid={Boolean(errors.region)}>
+              <FormLabel color="grey.400">Region</FormLabel>
+              <Input
+                {...register("region", { required: "Please enter a region" })}
+              />
+              <FormErrorMessage> {errors.region?.message} </FormErrorMessage>
+            </FormControl>
+          </HStack>
+        </FormControl>
+
+        <Button onClick={handleSubmit(onSubmit, onError)}>Submit</Button>
+      </VStack>
+    </Box>
   );
 };
 
```

Comment:
could you move where `CreateAssessmentPage` to `AdminDashboard` so that it does not have a navbar? I think we discussed this earlier on Slack! i.e you can follow the example used for the `QuestionPage` [here](https://github.com/uwblueprint/jump-math/blob/9292ef89570eef149c8072ccd90d783080033fb1/frontend/src/components/pages/admin/AdminDashboard.tsx#L24).